SpatialGDK/Source/SpatialGDKFunctionalTests/Public/SpatialFunctionalTestRequireHandler.h

```diff
@@ -11,7 +11,7 @@ struct FSpatialFunctionalTestRequire
 {
 	FString Msg;
 	bool bPassed;
-	FString ErrorMsg;
+	FString StatusMsg;
```

Comment:
Is this used elsewhere by any chance, or only here?

Reply:
Looks to me like it's only here